README.md

```diff
@@ -1 +1,62 @@
-# gluster-ansible-maintenance
\ No newline at end of file
+Role Name
+=========
+
+replace_node - Replace gluster node with a new node.
+
+Requirements
+------------
+
+- Ansible >= 2.6
+- lvm2 (Optional)
+
+
+NOTE: Inventory should contain a single node, which should be part of the cluster. This node should be different than any of the three variables explained below, this requirement is to collect the right peers for the new node.
+
+Role Variables
+--------------
+
+| Name                     |Choices| Default value         | Comments                          |
+|--------------------------|-------|-----------------------|-----------------------------------|
+| gluster_maintenance_old_node |    | UNDEF   | The node which has to be replaced with a new node. Just the node name is needed to get the peer id, the node need not be accessible. Provide the name that was used to probe the peers. |
+| gluster_maintenance_new_node |   | UNDEF | New node which will replace the old node. This name can be same as the old node or different name. |
+| gluster_maintenance_cluster_node |  | UNDEF | The node on which the peer, volume-id details are collected. This node should be part of the trusted storage pool. And should be different from old_node or new_node. |
```

Comment:
This node should be part of the trusted storage pool. And should be different from old_node or new_node. -->
This node should be part of the trusted storage pool, and different from the node being replaced

Reply:
gluster_maintenance_cluster_node or gluster_maintenance_peer_node ?

Reply:
It was named cluster_node since peer node could still not be part of cluster till added.